packages/contracts/contracts/rewards/IRewardsIssuer.sol

```diff
@@ -10,13 +10,20 @@ interface IRewardsIssuer {
      * @return subgraphDeploymentId Subgraph deployment id for the allocation
      * @return tokens Amount of allocated tokens
      * @return accRewardsPerAllocatedToken Rewards snapshot
+     * @return accRewardsPending Tokens pending to be claimed
```

Comment:
nit: the description here isn't super clear. Maybe something like "Snapshot of accumulated rewards from previous allocation resizing, pending to be claimed"?